project-skeletons/elisp-package-with-docs/Makefile

```diff
@@ -58,3 +58,12 @@ $(TEXI_MANUAL) : $(PKG_DIR) $(ORG_MANUAL)
 	$(CASK) exec $(EMACSBATCH) \
 	-l org -l ox-texinfo \
 	--file=$(ORG_MANUAL) -f org-texinfo-export-to-texinfo
+
+
+test: unit ecukes
+
+unit:
```

Comment:
This has a dependency on Cask being initialised, otherwise it errors on a fresh project. You'll need to add `$(PKG_DIR)` as a task dependency.